<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forget_widget</class>
 <widget class="QWidget" name="forget_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#confirm_pushButton
{
	border-image:url(:/pic/pic/button_login_normal.png)0 0 0 0 strecth strecth
}
QPushButton#confirm_pushButton:hover
{
	border-image:url(:/pic/pic/button_login_hover.png)0 0 0 0 strecth strecth
}
QPushButton#confirm_pushButton:pressed
{
	border-image:url(:/pic/pic/button_login_down.png)0 0 0 0 strecth strecth
}
QPushButton#cancel_pushButton
{
	border-image:url(:/pic/pic/button_login_normal.png)0 0 0 0 strecth strecth
}
QPushButton#cancel_pushButton:hover
{
	border-image:url(:/pic/pic/button_login_hover.png)0 0 0 0 strecth strecth
}
QPushButton#cancel_pushButton:pressed
{
	border-image:url(:/pic/pic/button_login_down.png)0 0 0 0 strecth strecth
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background-color: rgb(121, 121, 121);
border-radius:20px  ;
}
QLineEdit{
border-radius:6px  ;
}
QPushButton{
background-color: rgb(242, 255, 210);
background-color: rgb(255, 170, 255);
font: 290 9pt &quot;Microsoft YaHei UI&quot;;	
border-radius:3px  ;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="acc_lineEdit">
        <property name="placeholderText">
         <string>账号</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="email_lineEdit">
        <property name="placeholderText">
         <string>联系方式</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="pass_lineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>新密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="pass2_lineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>确认密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="confirm_pushButton">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_pushButton">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
